spec/views/labware/show_html_erb_spec.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'labware/show.html.erb', type: :view do
```

Comment:
Test could also include 'when rendering a plate' to cover all possibilities?

Reply:
Yes added now